parley/src/layout/line/greedy.rs

```diff
@@ -523,7 +523,6 @@ impl<'a, B: Brush> BreakLines<'a, B> {
                         // Compute the run's vertical metrics
                         line.metrics.ascent = line.metrics.ascent.max(run.metrics.ascent);
                         line.metrics.descent = line.metrics.descent.max(run.metrics.descent);
-                        line.metrics.leading = line.metrics.leading.max(run.metrics.leading);
```

Comment:
Why aren't we calculating the line's leading any more? Unclear to me.

Reply:
Ah, I probably should've mentioned this in the PR itself so more people could take a look at it. I believe that this is dead code now, since leading is always calculated [here](https://github.com/linebender/parley/blob/5be9994032e446bb9fbb893324631329e3e1f1f3/parley/src/layout/line/greedy.rs#L607-L608), and both of the removed leading-calculation lines should flow into that code unconditionally.

Reply:
Thanks and I think you're correct. Good catch!